src/AstroImages.jl

```diff
@@ -47,31 +49,43 @@ for n in (8, 16, 32, 64)
     end
 end
 
-struct AstroImage{T<:Real,C<:Color}
-    data::Matrix{T}
+struct AstroImage{T<:Real,C<:Color, N}
+    data::NTuple{N, Matrix{T}}
 end
 
 """
     AstroImage([color=Gray,] data::Matrix{Real})
+    AstroImage(color::Type{<:Color}, data::NTuple{N, Matrix{T}}) where {T<:Real, N}
 
 Construct an `AstroImage` object of `data`, using `color` as color map, `Gray` by default.
 """
 AstroImage(color::Type{<:Color}, data::Matrix{T}) where {T<:Real} =
-    AstroImage{T,color}(data)
-AstroImage(data::Matrix{T}) where {T<:Real} = AstroImage{T,Gray}(data)
+    AstroImage{T,color, 1}(ntuple(i-> data, 1))
```

Comment:
No need to use `ntuple` (which doesn't have a good performance, unfortunately) when you know the number, just write out the tuple
```suggestion
    AstroImage{T,color,1}((data,))
```